lib/terraform_landscape/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # Defines the gem version.
 module TerraformLandscape
-  VERSION = '0.1.18'.freeze
+  VERSION = '0.1.19'.freeze
```

Comment:
Leave this out. We'll bump the version number in a separate diff.

Reply:
Yeah, I realized this after I did it (this was so I could basically use it locally for my personal uses :) )